lib/repl.js

```diff
@@ -315,7 +315,8 @@ function REPLServer(prompt,
       } catch (e) {
         err = e;
 
-        if (err && err.message === 'Script execution interrupted.') {
+        if (err && err.message ===
+          errors.message('ERR_SCRIPT_EXECUTION_INTERRUPTED')) {
```

Comment:
I think you can check if `err.code === 'ERR_SCRIPT_EXECUTION_INTERRUPTED'` here instead (which is kinda the point of this PR, the code instead of the message should be used to match a type of error)